mne/event.py

```diff
@@ -229,8 +229,10 @@ def read_events(filename, include=None, exclude=None, mask=None,
 
     Returns
     -------
-    events: array, shape (n_events, 3)
-        The list of events.
+    events : array, shape (n_events, 3)
+        The array of events. The first column contains the event time in
+        samples, with ``first_samp`` included. The second column contains
+        [...]. The third column contains the event id.
```

Comment:
the second column was the *previous* value of the stim channel before this new onset, and they usually don't use it. +1 for just talking about the first and third like you do here, and telling people to look at
```
:term:`events`
```
for more information, then adding stuff to the `glossary.rst`

Reply:
So something along the lines of:

```
The array of events. The first column contains the event time in
samples, with ``first_samp`` included. The third column contains 
the event id. More information about :term:`events`.
```

Reply:
Yep!